fix(css): background-clip update border-box prose (#35884)
mdn/content: files/en-us/web/css/background-clip/index.md

{{EmbedInteractiveExample("pages/css/background-clip.html")}}

> The background is always drawn behind the border, so `background-clip: border-box` has visual effect only when the border is partially opaque or has transparent or partially opaque regions. Also, it makes no sense to have `background-clip: text` if the text is not transparent or partially opaque.

Reviewer: ```suggestion
The background is always drawn behind the border, so `background-clip: border-box` has a visual effect only when the border is partially opaque or has transparent or partially opaque regions. Also, the `background-clip: text` property has little to no visual effect if the text is fully or partially opaque.
```
the "no sense" may be interpreted as judgemental (don't want to infer that people have no sense (though they might not 😝)
